app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
There is no reason to have all of these devise views in your codebase if you do not change any of them. For instance, you added a `:username` to the User model, so it is understandable that you will have the devise view for signing up so you can add an input field for the username. But for the login in it is not necessary to have the devise view in the codebase since you are not making any changes to it.